lib/elixir/test/elixir/protocol_test.exs

```diff
@@ -387,7 +394,9 @@ defmodule Protocol.ConsolidationTest do
 
   test "consolidation updates attributes" do
     assert Sample.__protocol__(:consolidated?)
+    assert Sample.__protocol__(:impls) == [ImplStruct]
```

Comment:
To be honest, I am not super OK with `{:ok, [ImplStruct]}` being returned though. Maybe `{:consolidated, module} | :not_consolidated`?